src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
 ```
 [This article](/how-to-distribute-signed-http-exchanges/) illustrates how to use nginx to distribute SXGs.
 
+## Unique Advantages
+
+SXG is one of many possible technologies to enable [cross-origin prefetching](https://developer.chrome.com/blog/cross-origin-prefetch/). When deciding which technology to use, you may need to trade off between optimizing different aspects. The following sections illustrate a few of the unique values that SXG provides in the space of possible solutions.
+
+### Fewer requests to serve
+
+With cross-site prefetching, your server may need to serve additional requests. This corresponds to cases where a page was prefetched, but either the user didn't visit the page, or the prefetched bytes couldn't be shown to the user. For SXG, this additional traffic can be significantly reduced:
+
+- SXGs are cached and may be sent to users until they expire.
+- SXGs can be shown to users with cookies on your site.
+
+### Page speed improvement
+
+You may see additional page speed improvement from the following factors:
+
+- SXGs can be shown to users with cookies for your site.
+- SXG can also prefetch subresources for your pages, such as JavaScript, CSS, fonts, and images.
+- In the near future, SXG prefetching from Google Search will be available on more surfaces, including desktop form factors and all Chromium browsers.
+
+{% Details %}
+{% DetailsSummary %}
+Long-term
+In the long term, other factors may influence which technology will provide the biggest improvement to page speed.
+{% endDetailsSummary %}
+
+- As more referrers implement cross-site prefetching for their outlinks, they may choose to implement only one technology, or multiple.
+- The reduced request serving for publishers could enable referrers to prefetch SXGs for links with lower predicted clickthrough rate.
+{% endDetails %}
+
+## Roadmap
+
+Google Search is continuing to improve the functionality of its SXG cache. Here are a couple upcoming improvements that address common limitations for sites looking to implement SXG.
```

Comment:
Does this section need to be framed in the context of Google Search? i.e. If another referrer (internal or external) were to adopt SXG, would these roadmap features not benefit them as well?

Reply:
OK, reframed the intro section to be more general. The specific two items here are current limitations of the Google SXG Cache, not of the SXG format. There are other potential improvements at the format level but they didn't make my top 2. :)